docs/CLI.md

```diff
@@ -350,6 +350,16 @@ The default regex matching works fine on small runs, but becomes slow if provide
 
 :::
 
+### `--seed=<num>`
+
+Sets a seed value that can be retrieved in a test file via [`jest.getSeed()`](JestObjectAPI.md#jestgetseed). If this option is not specified Jest will randomly generate the value.
+
+The seed value must be between `-0x80000000` and `0x7fffffff` inclusive.
+
+```bash
+jest --seed=1324
+```
+
```

Comment:
`--showSeed` documentation is missing. Please place it in alphabetical order after `--showConfig`:

```suggestion

### `--showSeed`
Prints the seed value in the test report summary. See [`--seed=<num>`](#seednum) for the details.
```